WeakAuras/WeakAuras.lua

```diff
@@ -2541,34 +2541,57 @@ function Private.AddMany(tbl, takeSnapshots)
 
   local order = loadOrder(tbl, idtable)
   coroutine.yield()
+
+  if takeSnapshots then
+    for _, data in ipairs(order) do
+      Private.Snapshot(data)
+      coroutine.yield()
+    end
+  end
+
   local groups = {}
+  local bads = {}
   for _, data in ipairs(order) do
-    WeakAuras.PreAdd(data)
-    if data.regionType == "dynamicgroup" or data.regionType == "group" then
-      groups[data] = true
+    if data.parent and bads[data.parent] then
+      bads[data.id] = true
```

Comment:
Honestly, i'm not super sure if this is the best idea, but I can't think of what else to do in this case. Current behavior with Add() is to just not send the bad data to pAdd(), which I think would spill the group's contents onto the screen in an ugly way.

Reply:
Not sure either, the print output is good though.